articles/portal/ptl-ref-portal-api.md

```diff
@@ -3678,6 +3678,119 @@ Returns a CSV report for the whole month specified including the date provided.
   Consumption (GB) | Average storage used over the month in gigabytes
   Cost | Amount that would be charged if this were for a whole month (in pounds)
 
+## GET /api/billing/billing-csv
+
+Returns a CSV containing billing data for the period given.
+
+### API version
+
+v1
+
+### Request
+
+#### Request body
+
+None
+
+#### Parameters
+
+None
+
+#### URI parameters
+
+  Parameter name | Description | Type | Mandatory (Default)
+  ---------------|-------------|------|--------------------
+  period | The date for which you require the monthly report in the format YYYY-MM | String | Y
+  org_id | The ID of the compute service for which you want the billing report | String | Y
+
+#### Example request (Curl)
+
+```bash
+curl -b /tmp/cookies.txt -X GET -H "Accept: application/json" -k https://portal.skyscapecloud.com/api/billing/billing-csv?period=2018-01-&org_id=xx-xx-xx-xxxxxxx
```

Comment:
is the "-" correct after the period and before the "&org_id"?

Reply:
Removed in 98e1e97